### PR TITLE
feat: adds ability to create multiple orgs under the same account

### PR DESCRIPTION
**Pull request by rtpa25**
# Description 📣

Fixes https://github.com/Infisical/infisical/issues/1082
[Screen recording example ](https://www.loom.com/share/c3b1ac852d724403904919552bed10d3)

## Type ✨

- [ ] Bug fix
- [x] New feature
- [ ] Breaking change
- [ ] Documentation

---

- [x] I have read the [contributing guide](https://infisical.com/docs/contributing/overview), agreed and acknowledged the [code of conduct](https://infisical.com/docs/contributing/code-of-conduct). 📝

### CODE REVIEW

**Comment by rtpa25** [comment]
@maidul98 @vmatsiiako awaiting review

File: frontend/src/views/Org/components/CreateOrgModal.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+import { Controller, useForm } from "react-hook-form";
+import { useRouter } from "next/router";
+import { yupResolver } from "@hookform/resolvers/yup";
+import * as yup from "yup";
+
+import { useNotificationContext } from "@app/components/context/Notifications/NotificationProvider";
+import { Button, FormControl, Input, Modal, ModalContent } from "@app/components/v2";
+import { useCreateOrg } from "@app/hooks/api";
+
+const schema = yup
```
Comment (akhilmhdh) [inline frontend/src/views/Org/components/CreateOrgModal.tsx:11]: Can you please change to zod. We are moving towards `zod` from yup.  As backend also uses zod we wanna keep one validation lib through out infisical

File: frontend/src/views/Org/components/CreateOrgModal.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            control={control}
+            defaultValue=""
+            name="name"
+            render={({ field, fieldState: { error } }) => (
+              <FormControl label="Name" isError={Boolean(error)} errorText={error?.message}>
+                <Input {...field} placeholder="Acme Corp" />
+              </FormControl>
+            )}
+          />
+          <Button
```
Comment (akhilmhdh) [inline frontend/src/views/Org/components/CreateOrgModal.tsx:86]: A cancel button is needed here

**Review by akhilmhdh — CHANGES_REQUESTED** [review]
Rest looks good to me. 

Although one que to @vmatsiiako @maidul98 @dangtony98 

Don't we need some kinda of restriction on number of org created;

**Comment by maidul98** [comment]
> Rest looks good to me.

Although one que to @vmatsiiako @maidul98 @dangtony98

Don't we need some kinda of restriction on number of org created;

For time being, I think we can leave it as is. @rtpa25 please address the comments made by Akhi so we can get this merged in

**Comment by rtpa25** [comment]
Will do it tom

**Review by akhilmhdh — APPROVED** [review]

**Pull request merged**